docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1184,7 +1188,7 @@ The global configuration contains non-tenant specific configuration parameters (
 
 `${server.config.dir}/config/default/fhir-server-config.json`
 
-```
+``` json
```

Comment:
the `json` hint seems to cause the github editor some trouble, since technically the snippet isn't valid json

Reply:
this gets compiled nicely into a stylized editor